np/numpy.js

```diff
@@ -0,0 +1 @@
/python3.8/site-packages/numpy/core/tests/test_ufunc.py",start:13371701,end:13441359,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/test_half.py",start:13441359,end:13459964,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/test_numeric.py",start:13459964,end:13562004,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/test_item_selection.py",start:13562004,end:13565603,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/test_arrayprint.py",start:13565603,end:13599449,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/test_einsum.py",start:13599449,end:13643590,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/test_function_base.py",start:13643590,end:13654932,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/__init__.py",start:13654932,end:13654932,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/test_print.py",start:13654932,end:13662071,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/test_machar.py",start:13662071,end:13663212,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/test_scalarmath.py",start:13663212,end:13690489,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/test_datetime.py",start:13690489,end:13786021,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/test_defchararray.py",start:13786021,end:13811648,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/test_abc.py",start:13811648,end:13814050,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/test_dtype.py",start:13814050,end:13845767,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/test_extint128.py",start:13845767,end:13851517,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/test_longdouble.py",start:13851517,end:13857831,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/test_scalarinherit.py",start:13857831,end:13859628,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/_locales.py",start:13859628,end:13861934,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/test_umath.py",start:13861934,end:13968741,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/test_memmap.py",start:13968741,end:13976057,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/test_regression.py",start:13976057,end:14061125,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/test_umath_complex.py",start:14061125,end:14080448,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/test_scalar_ctors.py",start:14080448,end:14082742,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/data/astype_copy.pkl",start:14082742,end:14083458,audio:0},{filename:"/lib/python3.8/site-packages/numpy/core/tests/data/recarray_from_file.fits",start:14083458,end:14092098,audio:0}],remote_package_size:7482064,package_uuid:"bc18a0cb-fd54-4271-bcf7-bf29a53ba511"})})();
```

Comment:
Is this available as an `npm` module? I'd rather not commit the blob of source to the repo if it's installable in another way.

Reply:
This is not available in `npm`. Will remove it for future pull requests.